README.md

```diff
@@ -512,17 +512,17 @@ This report is autogenerated and includes tokenizers and detokenizers tests. The
   <tbody>
     <tr>
       <td >BPE</td>
-      <td >97.18</td>
+      <td >95.69</td>
       <td >4544</td>
     </tr>
     <tr>
       <td >SentencePiece</td>
-      <td >89.19</td>
+      <td >86.36</td>
```

Comment:
why usage of official operations decreases pass rate?

Reply:
There is a bug in string tensor unpack: the bugs effect is that when input string is empty then output tensor is the last tensor which is produced by str upack.

I will create a bug for that.
![image](https://github.com/user-attachments/assets/3f2f654e-1d4a-4b69-92fe-366a20e28d20)

Reply:
Created CVS-159636 ticket for that

Reply:
I suppose we should wait for this bug to be fixed? before merging this PR

Reply:
I don't see much harm if we merge it how, but i'm not sure. @apaniukov how do you think?

Reply:
why do we need to merge code which drops pass rate?

Reply:
Maybe it is better to wait, but not for long.